.eslintrc.js

```diff
@@ -22,5 +22,6 @@ module.exports = {
     'space-before-function-paren': ['error', 'never'],
     'comma-dangle': ['error', 'only-multiline'],
     'object-property-newline': ['error', { allowAllPropertiesOnSameLine: false }],
+    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
What do you think about , just desirable in (eslint) in the line that it request, and then in some time, try to find the correct type. Give your opinion about it